website/source/docs/agent/options.html.markdown

```diff
@@ -339,6 +349,98 @@ The remaining keys in the list are considered secondary and are used for
 decryption only. During message decryption, Serf uses the configured encryption
 keys in the order they appear in the keyring file until all keys are exhausted.
 
+#### Memberlist Profile Configuration Key Reference
+
+The following configuration keys may be used in a configuration file referenced
+using the `profile_path` option in Serf configuration.
+
+Options with a `timeout`, `time` or `interval` suffix may be expressed using
+common unit suffixes for time units - for example `500ms`, `10s` and `1m`.
+  
+* `awareness_max_mult` - Increases the probe interval if the node becomes aware
+  that it might be degraded and not meeting the soft real time requirements to
+  reliably probe other nodes.
+
+* `gossip_interval` - The interval between sending messages that need to be 
+  gossiped that haven't been able to piggyback on probing messages. If this is set
+  to zero, non-piggyback gossip is disabled. By lowering this value (more frequent)
+  gossip messages are propagated across the cluster more quickly at the expense of
+  increased bandwidth.
+  
+* `gossip_nodes` - The number of random nodes to send gossip messages to per 
+  `GossipInterval`. Increasing this number causes the gossip messages to propagate
+  across the cluster more quickly at the expense of increased bandwidth.
+
+* `gossip_to_the_dead_time` - the interval after which a node has died that
+  we will still try to gossip to it. This gives the node a chance to refute.
+
+* `indirect_checks` - the number of nodes that will be asked to perform
```

Comment:
`- the..` should be capitalized, same for the above

Reply:
:+1: